src/components/Dashboard/Content/index.tsx

```diff
@@ -47,22 +48,26 @@ const DashboardContent = ({
       </div>
     </div>
   ) : (
-    <div className="flex flex-col h-full" data-test-id={dashboardDataTestIds.content}>
+    <div className="flex flex-col h-full" data-test-id={dashboardContentDataTestIds.base}>
       <div
-        style={{ backgroundImage: `url(${getHighResolutionImage(surveys[currentPosition].coverImageUrl)})` }}
+        style={{
+          backgroundImage: `url(${surveys.length ? getHighResolutionImage(surveys[currentPosition].coverImageUrl) : ''})`,
+        }}
         className="w-full h-[302px] rounded-[12px] bg-cover duration-500 ease-in-out"
       ></div>
       <div className="flex flex-row justify-between mt-[38px]">
         <div className="flex flex-col justify-between">
           <p className="text-white text-x-regular font-extrabold">{surveys[currentPosition].title}</p>
           <p className="text-white text-regular tracking-survey-tight opacity-60 mt-2">{surveys[currentPosition].description}</p>
         </div>
-        <button
-          type="button"
-          className="w-[56px] h-[56px] bg-white rounded-full inline-flex items-center justify-center text-black-chinese"
-        >
-          <ArrowRight />
-        </button>
+        <Link to={`surveys/${surveys[currentPosition].id}`}>
+          <button
+            type="button"
+            className="w-[56px] h-[56px] bg-white rounded-full inline-flex items-center justify-center text-black-chinese"
```

Comment:
Why do we need `inline-flex`?

Reply:
@nvminhtue if we don't set `inline-flex`, the arrow position will be incorrect
![image](https://github.com/manh-t/react-survey/assets/60863885/3e8bd7e1-55b8-489d-a2f6-33efb1a534a7)